packages/user-group-service/src/users/typedef.graphql

```diff
@@ -154,46 +88,36 @@ type Mutation {
   """
   Add a new User
   """
-  addUser(input: UserInput): UserType
+  addUser(input: RoverUserInput): RoverUserType
```

Comment:
We should standardise the GraphQL type names. Make it `User` to align it with `Group` and make it easier to find and understand for anyone.

```suggestion
  addUser(input: UserInput): User
```

Reply:
Done